src/index.js

```diff
@@ -32,4 +32,29 @@ refreshBtn.addEventListener('click', () => {
     });
 });
 
+const loadScores = () => {
+  const theader = document.createElement('tr');
+  const theaderPlayer = document.createElement('td');
+  theaderPlayer.innerHTML = 'Player name'
+  const theaderScore = document.createElement('td');
+  theaderScore.innerHTML = 'Score'
+  theader.appendChild(theaderPlayer);
+  theader.appendChild(theaderScore);
+  displayScores.appendChild(theader);
+  getScores()
+    .then((footballer) => {
+      footballer.result.forEach((playerscore) => {
+        const scoreEntryElem = document.createElement('tr');
+        const playerNameElem = document.createElement('td');
+        playerNameElem.innerHTML = `${playerscore.user}`;
+        scoreEntryElem.appendChild(playerNameElem);
+        const playerScoreElem = document.createElement('td');
+        playerScoreElem.innerHTML = `${playerscore.score}`;
+        scoreEntryElem.appendChild(playerScoreElem);
+        displayScores.appendChild(scoreEntryElem);
+      })
+    })
+}
+
 gameID('foorball');
+window.onload(loadScores());
```

Comment:
- [ ] For some reason there are errors preventing the app from working properly. Kindly fix them.
- [ ] The live demo is also not working and it has a different background. I don't know if you haven't updated it. 
![Screenshot from 2021-08-06 22-57-41](https://user-images.githubusercontent.com/64894927/128545877-ee81e330-fde6-4ec5-9a27-c0098dd0f17e.png)       
![Screenshot from 2021-08-06 22-54-32](https://user-images.githubusercontent.com/64894927/128545934-d33ef178-b1e1-4db3-ae96-b21d23faa9ca.png)

Reply:
I have resolved the 2 errors mentioned:
- app not opening
- live link